components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -410,6 +411,11 @@ class GeckoEngineSession(
         }
     }
 
+    override fun captureThumbnail(): Bitmap? {
+        // TODO Waiting for the GeckoView to create an API for this
```

Comment:
nit: can we add the GV bug to this TODO (https://bugzilla.mozilla.org/show_bug.cgi?id=1462018). We should avoid empty TODOs and always link to a bug.

Reply:
👍